### PR TITLE
Resample bvals

### PR DESCRIPTION
**Pull request by GuillaumeTh**
Follow up of PR #367. Centroids are know sorted and the bvec file no longer required.

### CODE REVIEW

File: scripts/scil_resample_bvals.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+    bvals, bvecs = read_bvals_bvecs(args.bvals, None)
+    # Find the volume indices that correspond to the shells to extract.
+    tol = args.tolerance
+
+    centroids, shell_indices = identify_shells(bvals, tol)
+    sort_index = np.argsort(centroids)
+    centroids = centroids[sort_index]
+    shell_indices = shell_indices[sort_index]
+
```
Comment (arnaudbore) [inline scripts/scil_resample_bvals.py:72]: Can you put sort=True as an option in identify_shells. Default=False

**Review by arnaudbore — APPROVED** [review]
LGTM need a test before merging.

**Review by mdesco — COMMENTED** [review]
Nothing works well... sorry to say.

scil_resample_bvals.py bval 0 1000 2000 newbals1 --tolerance 20
scil_resample_bvals.py bval 0 1000 2000 newbals2 --tolerance 200
scil_resample_bvals.py bval 0 1000 2000 newbals3 --tolerance 200

**Review by mdesco — COMMENTED** [review]
Why is failing if tolerance is <= 65? Could just give a warning, no? why the crash and I don't understand the error message.

**Comment by mdesco** [comment]
If I try something like this: 
scil_resample_bvals.py bval 0 1000 1950 new2 --tolerance 70 -f

The output is also wrong, no?

**Review by mdesco — APPROVED** [review]

File: scripts/scil_resample_bvals.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                             assert_inputs_exist, assert_outputs_exist)
+from scilpy.utils.bvec_bval_tools import identify_shells
+
+
+def _build_arg_parser():
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+
+    parser.add_argument('bvals',
```
Comment (arnaudbore) [inline scripts/scil_resample_bvals.py:32]: Please:
in_bval

out_bval

File: scripts/scil_resample_bvals.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    if args.verbose:
+        logging.basicConfig(level=logging.INFO)
+
+    assert_inputs_exist(parser, [args.bvals])
```
Comment (arnaudbore) [inline scripts/scil_resample_bvals.py:61]: don't need [ ] same for assert_outputs_exist

**Pull request merged**